openshift/Makefile

```diff
@@ -14,9 +14,11 @@ REQUIREMENTS_IMG ?= pip-requirements
 
 .PHONY: build-requirements
 build-requirements:
-	cp ../images/ansible-operator/Pipfile* .
+	mkdir -p ./images/ansible-operator/
+	cp ../images/ansible-operator/Pipfile* ./images/ansible-operator/
```

Comment:
Why is this copy required, if the `openshift/Dockerfile.requirements` can directly copy the `Pipfile*` files from the upstream dir?